src/fdk_rdf_parser/classes/info_model.py

```diff
@@ -63,6 +63,12 @@ def add_model_element(self: Any, element: ModelElement) -> Any:
 
         self.add_subject_from_element_or_property(element.subject)
 
+        if element.codes:
+            [
+                self.add_subject_from_element_or_property(code.subject)
```

Comment:
Er nok bedre å skille code_element fra resten
dvs opprett containsCodeElements som felt på info_model og lag en egen `add_code_element_from_element_or_property`

Reply:
Vet ikke helt om jeg skjønner hva du mener her - grunnen til at jeg går gjennom code elements er for å få begrepene de referer til med i containsSubjects, slik at de kan vises riktig i portalen @NilsOveTen

Reply:
Ah, ja. Misforstod hva som skjedde her, good shit it is :+1: